client/components/status/status.js

```diff
@@ -29,10 +32,18 @@ export default class Status extends React.Component {
       .then(response => {
         if (response.data["control:Auth-Type"] !== "Accept") {
           logout(cookies, orgSlug);
+          toast.error(loginError, {
+            onOpen: () => toast.dismiss(mainToastId),
+          });
+          logError(response, '"control:Auth-Type" !== "Accept"');
         }
       })
-      .catch(() => {
+      .catch((error) => {
         logout(cookies, orgSlug);
+        toast.error(loginError, {
+          onOpen: () => toast.dismiss(mainToastId),
+        });
```

Comment:
Added this code, so success message will automatically be closed when the token check error occurred